Assets/MixedRealityToolkit/InputSystem/Focus/FocusProvider.cs

```diff
@@ -377,7 +377,7 @@ public uint GenerateNewPointerId()
         /// </summary>
         public void UpdateCanvasEventSystems()
         {
-            Debug.Assert(UIRaycastCamera != null, "You must assign a UIRaycastCamera on the FocusProvider before updating your canvases.");
+             Debug.Assert(UIRaycastCamera != null, "You must assign a UIRaycastCamera on the FocusProvider before updating your canvases.");
```

Comment:
Since this is an assert, i assume that UpdateCanvasEventSystems is not always called for an app?